src/App.tsx

```
@@ -1,38 +1,73 @@
+          onClick={toggleTheme}
+          aria-label={`Switch to ${theme === "dark" ? "light" : "dark"} mode`}
+        >
+          {theme === "dark" ? "🌙 Dark" : "☀️ Light"}
+        </button>
+      </div>
+
+      <h1>Vite + React</h1>
+
+      <div className="card">
+        <button onClick={() => setCount((c) => c + 1)}>
+          count is {count}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)